pkgs/applications/editors/neovim/gnvim/wrapper.nix

```diff
@@ -14,6 +14,7 @@ stdenv.mkDerivation {
   '' else ''
     makeWrapper '${gnvim-unwrapped}/bin/gnvim' "$out/bin/gnvim" \
       --prefix PATH : "${neovim}/bin" \
+      --set GDK_PIXBUF_MODULE_FILE "${librsvg.out}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache" \
```

Comment:
Please do not hardcode versions. This will likely cause break in future updates.